client/src/session/rest/api/common.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright © 2022, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
+// Licensed under SAS Code Extension Terms, available at Code_Extension_Agreement.pdf
+
+/**
+ * Compute
+ * The Compute API defines the replacement for the application servers that were used for SAS 9. The Compute API enables clients to perform the following tasks: <UL> <LI>Submit and execute SAS code. <LI>Obtain the SAS log or listing information from executed SAS code. <LI>View output that is generated by a SAS code submission. <LI>Access SAS data sets that are created by a SAS code submission. <LI>Access SAS filerefs that are created by a SAS code submission. <LI>Access SAS variables that are defined and used in a SAS code submission. <LI>Retrieve information about engines that are available in a session. </UL>
+ *
+ * The version of the OpenAPI document: 11
+ * Contact: devex@sas.com
+ *
+ * NOTE: This class is auto generated by OpenAPI
```

Comment:
If these are auto-generated (and thus, shouldn't be reviewed), then we may want to suppress them w/ a `.gitattributes` file

Reply:
Potential `.gitattributes` file...
```
client/src/session/rest/api/common.ts linguist-generated=true
client/src/session/rest/api/compute.ts linguist-generated=true
client/src/session/rest/api/configuration.ts linguist-generated=true
```
Are there others that should be ignored, @scnwwu ?